safe/definitions/default_settings.py

```diff
@@ -23,7 +23,7 @@
     'ISO19115_LICENSE': 'Free use with accreditation',
 
     'keywordCachePath': abspath(join(
-        expanduser('~'), '.inasafe', 'keywords.db')),
+        expanduser('~'), 'inasafe', 'metadata.db')),
```

Comment:
Why do you remove the hidden folder ?
Anyway, we shouldn't use a folder in the user directory. We should use the QGIS user directory